libyahoo-plusplus.c

```diff
@@ -469,7 +469,7 @@ yahoo_fetch_url(YahooAccount *ya, const gchar *url, const gchar *postdata, Yahoo
 		} else {
 			g_string_append(headers, "Content-Type: application/x-www-form-urlencoded\r\n");
 		}
-		g_string_append_printf(headers, "Content-Length: %d\r\n", strlen(postdata));
+		g_string_append_printf(headers, "Content-Length: %ld\r\n", strlen(postdata));

```

Comment:
This one causes compile errors on 32-bit systems.

If you could switch this to use `G_GSIZE_FORMAT` instead of `ld`, then it would compile cleanly everywhere